hail/src/main/scala/is/hail/asm4s/joinpoint/ParameterPack.scala

```diff
@@ -216,9 +216,10 @@ case class ParameterStoreArray(pss: IndexedSeq[ParameterStore[_]]) extends Param
 
 }
 
-case class ParameterPackTriplet[P](ti: TypeInfo[P]) extends ParameterPack[TypedTriplet[P]] {
+case class ParameterPackTriplet[P](t: PType) extends ParameterPack[TypedTriplet[P]] {
+  val ti: TypeInfo[P] = ir.typeToTypeInfo(t).asInstanceOf[TypeInfo[P]]
   val ppm = implicitly[ParameterPack[Code[Boolean]]]
-  val ppv = ParameterPack.code(ti)
+  val ppv = ParameterPack.code[P](ti).asInstanceOf[ParameterPack[Code[_]]]
```

Comment:
Do we need this change (the case to ParameterPack[Code[_]] from ParameterPack[Code[P])

Reply:
Yes, because ParameterPack is not variant.  I'd say don't be too concerned about this ParameterPack/TypedTriplet business.  It's all going away soon.